buildroot-v86/board/v86/linux.config

```diff
@@ -77,7 +77,7 @@ CONFIG_NET=y
 CONFIG_PACKET=y
 CONFIG_UNIX=y
 CONFIG_XFRM_USER=y
-# CONFIG_INET is not set. Since we won't be using the internet. Reduces the size by 400kb.
+CONFIG_INET is not set. Since we won't be using the internet. Reduces the size by 400kb.
```

Comment:
Should be `CONFIG_INET=y` with nothing after it.  Did you test this, because this won't build.